apps/evm/src/pages/Account/NewPage/Transactions/TransactionRow/index.tsx

```diff
@@ -0,0 +1,103 @@
+      return 'transactionIn';
+    case TxType.Borrow:
+    case TxType.Redeem:
+      return 'transactionOut';
+    default:
+      return 'transactionCollateral';
+  }
+};
+
+export const TransactionRow: React.FC<TransactionRowProps> = ({ transactionData, className }) => {
+  const { transaction, asset } = transactionData;
+  const { underlyingToken } = asset.vToken;
+  const { t } = useTranslation();
+
+  const transactionTitle = useMemo(() => {
+    switch (transaction.txType) {
+      case TxType.Mint:
+        return t('account.transactions.txType.mint');
+      case TxType.Repay:
+        return t('account.transactions.txType.repay');
+      case TxType.Borrow:
+        return t('account.transactions.txType.borrow');
+      case TxType.Redeem:
+        return t('account.transactions.txType.redeem');
+      case TxType.Approve:
+        return t('account.transactions.txType.approve');
+      case TxType.ExitMarket:
+        return t('account.transactions.txType.exitMarket');
+      default:
+        return t('account.transactions.txType.enterMarket');
+    }
+  }, [t, transaction]);
+
+  return (
+    <li
+      className={cn(
+        'flex flex-col px-4 py-3 md:p-0 bg-cards rounded-xl md:rounded-none w-full md:flex-row items-center justify-evenly',
+        className,
+      )}
+    >
+      <a
+        className="flex flex-col md:flex-row w-full"
+        href={generateExplorerUrl({
+          hash: transaction.txHash,
+          urlType: 'tx',
+          chainId: transaction.chainId,
+        })}
+        target="_blank"
+        rel="noreferrer"
+      >
+        <div className="flex w-full md:w-70 items-center">
+          <div className="p-[6px] md:p-2 mr-2 md:mr-3 flex items-center justify-center bg-lightGrey rounded-full">
+            <Icon className="h-3 w-3 md:h-4 md:w-4" name={getTransactionIcon(transaction.txType)} />
+          </div>
+          <div className="flex flex-row w-full md:w-auto items-center md:items-start justify-between md:flex-col">
+            <span className="text-sm">{transactionTitle}</span>
+            <span className="text-grey text-xs">{format(transaction.txTimestamp, 'hh:mm aa')}</span>
+          </div>
+        </div>
+
+        <hr className="block w-full text-lightGrey my-3 md:hidden" />
+
+        <div className="flex w-full items-center">
+          <TokenIcon className="mr-2 self-start" token={underlyingToken} />
+          <TransactionDetails className="flex-1" transactionData={transactionData} />
+          <a
+            className="hidden md:flex items-center self-start text-grey space-x-1 mt-1"
+            href={generateExplorerUrl({
+              hash: transaction.txHash,
+              urlType: 'tx',
+              chainId: transaction.chainId,
+            })}
+            target="_blank"
+            rel="noreferrer"
+          >
+            <span className="hidden lg:block text-sm underline">
+              {t('account.transactions.view')}
+            </span>
```

Comment:
When the component is hovered on and md and up, this text should be white. You can use the group feature from Tailwind to detect that the root tag is hovered and apply styling to this element in consequence: https://v3.tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state